app/components/Map.tsx

```diff
@@ -63,59 +76,229 @@ function circleDiameter(count: number, total: number): number {
   return minSize + (count / total) * (maxSize - minSize);
 }
 
-function addMarkers(observations: Observation[], map: mapboxgl.Map) {
+function makeStationMarker(
+  groupObservations: Observation[],
+  species: _.Collection<string>,
+  maxSpecies: number,
+  setInspectedObservations: (observations: Observation[]) => void
+) {
+  // Observations from a single station should have approximately identical
+  // coordinates, so we can pick any.
+  const firstObservation = groupObservations[0];
+  const coordinates = getObservationCoordinates(firstObservation);
+  const count = groupObservations.length;
+  const { station } = firstObservation;
+
+  const diameter = circleDiameter(species.size(), maxSpecies);
+  const thumbnailSize = diameter / 2;
+  const thumbnailOffset = thumbnailSize / 2;
+  const maxPreviewPhotosCount = 3;
+
+  const el = document.createElement('div');
+  el.className = 'marker';
+  el.setAttribute('style', `width: ${diameter}px; height: ${diameter}px;`);
+  el.innerHTML = `<img
+          src="${firstObservation.path}"
+          style="width: ${thumbnailSize}px; height: ${thumbnailSize}px; margin-top: -${thumbnailOffset}px; margin-left: -${thumbnailOffset}px;">`;
```

Comment:
It seems hacky that we are generating HTML by string formatting here. Could we document the reason or use JSX syntax instead, similar to how we generate the popup?